cli/tests/unit/README.md

```diff
@@ -1,78 +0,0 @@
-# Deno runtime tests
-
-Files in this directory are unit tests for Deno runtime.
-
-Testing Deno runtime code requires checking API under different runtime
-permissions (ie. running with different `--allow-*` flags). To accomplish this
-all tests exercised are created using `unitTest()` function.
-
-```ts
-import { unitTest } from "./test_util.ts";
-
-unitTest(function simpleTestFn(): void {
```

Comment:
I'd be great to keep this README file, it's especially useful to look up command to run a single test. Please retain it and update the examples.